gtk4/src/subclass/scrollable.rs

```diff
@@ -0,0 +1,36 @@
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::Cast;
+use gtk_sys;
+
+use Border;
+use Scrollable;
+
+pub trait ScrollableImpl: ObjectImpl {
+    fn get_border(&self, scrollable: &Self::Type, border: &mut Border) -> bool;
```

Comment:
Should this return an `Option<Border>` maybe?

Reply:
I did an attempt at this, not sure if the code looks that good

Reply:
I think that looks much better, but there's a bug in it :)